org.eclipse.xtext/src/org/eclipse/xtext/build/IncrementalBuilder.java

```diff
@@ -318,7 +318,7 @@ protected IResourceDescription getSerializableResourceDescription(IResourceDescr
 			return SerializableResourceDescription.createCopy(description);
 		}
 	
-		private IResourceServiceProvider getResourceServiceProvider(Resource resource) {
+		protected IResourceServiceProvider getResourceServiceProvider(Resource resource) {
```

Comment:
can you add please a since annotation

Reply:
Maybe make the indexer and operationCancelManager available to subtypes, too

Reply:
done